game-porting-toolkit/Portfile

```
@@ -0,0 +1,252 @@
+    depends_lib-append      port:gstreamer1 port:gstreamer1-gst-plugins-base port:gstreamer1-gst-plugins-good \
+        port:gstreamer1-gst-plugins-bad port:gstreamer1-gst-plugins-ugly port:gstreamer1-gst-libav
+    configure.args-replace  --without-gstreamer --with-gstreamer
+}
+
+default_variants            +gstreamer
+
+triplet.add_build           cross
+triplet.add_host            none
+
+configure.cc                ${prefix}/libexec/llvm-cx/bin/clang
+configure.cxx               ${prefix}/libexec/llvm-cx/bin/clang++
+configure.cmd               ${worksrcpath}/configure
+configure.compiler.add_deps no
+compiler.limit_flags        yes
+muniversal.arch_flag        no
+muniversal.arch_compiler    yes
+
+configure.ldflags-delete    -L${compiler.library_path}
+configure.optflags          -O3
+
+# Common compiler flags for both Mach-O and PE binaries.
+configure.cflags-append \
+    -Wno-deprecated-declarations \
+    -Wno-format \
+    -Wno-incompatible-pointer-types \
+    -Wno-implicit-function-declaration \
+    -Wno-int-conversion
+
+configure.env-append        CROSSCFLAGS=${configure.cflags}
+
+# Were only installing wine not the development files
+destroot.target             install-lib
+
+configure {
+    set configure.dir ${worksrcpath}-x86_64
+    portconfigure::configure_main
+
+    configure.args.x86_64
+    set configure.dir ${worksrcpath}-i386
+    configure.args-delete --enable-win64
+    configure.args ${configure.pre_args.i386} ${configure.args} ${configure.args.i386}
+    portconfigure::configure_main
+}
+
+build {
+    set build.dir ${worksrcpath}-x86_64
+    portbuild::build_main
+
+    set build.dir ${worksrcpath}-i386
+    portbuild::build_main
+}
+
+destroot {
+    set destroot.dir ${worksrcpath}-x86_64
+    portdestroot::destroot_main
+
+    set destroot.dir ${worksrcpath}-i386
+    portdestroot::destroot_main
+}
+
+post-destroot {
+    ln -sf ${prefix}/share/wine/gecko ${destroot}${prefix}/libexec/${name}/share/wine/gecko
+    ln -sf ${prefix}/share/wine/mono ${destroot}${prefix}/libexec/${name}/share/wine/mono
+
+    file copy ${destroot}${prefix}/libexec/${name}/bin/wine64           ${destroot}${prefix}/libexec/${name}/bin/wine
+    file copy ${destroot}${prefix}/libexec/${name}/bin/wine64-preloader ${destroot}${prefix}/libexec/${name}/bin/wine-preloader
+
+    file copy ${filespath}/gameportingtoolkit           ${destroot}${prefix}/bin/gameportingtoolkit
+    file copy ${filespath}/gameportingtoolkit-no-esync  ${destroot}${prefix}/bin/gameportingtoolkit-no-esync
+    file copy ${filespath}/gameportingtoolkit-no-hud    ${destroot}${prefix}/bin/gameportingtoolkit-no-hud
+
+    reinplace -q "s|@@PREFIX@@|${prefix}|g"             ${destroot}${prefix}/bin/gameportingtoolkit \
+                                                        ${destroot}${prefix}/bin/gameportingtoolkit-no-esync \
+                                                        ${destroot}${prefix}/bin/gameportingtoolkit-no-hud
```

The `-q` flag should only be used when you want MacPorts not to notify you if a no replacement took place. The only reason you might want to do this is if you are batch-processing all files of a particular type and not all of them need the replacement; that's not the case here as you're listing three specific files.

Can be simplified to:

```suggestion
    reinplace -W ${destroot}${prefix}/bin "s|@@PREFIX@@|${prefix}|g" \
        gameportingtoolkit \
        gameportingtoolkit-no-esync \
        gameportingtoolkit-no-hud
```